### PR TITLE
Update README to reactphp/http v1.0.0

### PR DESCRIPTION
**Pull request by SimonFrings**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -315,14 +315,20 @@ $connector->connect('tls://smtp.googlemail.com:465')->then(function (ConnectionI
...
-If you want to issue HTTP requests, you can add a dependency for
-[clue/reactphp-buzz](https://github.com/clue/reactphp-buzz).
-It can interact with this library by issuing all
-[HTTP requests through an SSH proxy server](https://github.com/clue/reactphp-buzz#ssh-proxy).
-When using the `SshSocksConnector` (recommended), this works for both plain HTTP
-and TLS-encrypted HTTPS requests. When using the `SshProcessConnector`, this only
-works for plaintext HTTP requests.
+This library also provides you with the usage of [HTTP requests through an SSH proxy server](https://github.com/reactphp/http#ssh-proxy).
+
+To send out HTTP requests you have to add a dependency for [ReactPHP's async HTTP client](https://github.com/reactphp/http). You can then use this HTTP client in combination with the `SshSocksConnector`. This works for both plain HTTP and TLS-encrypted HTTPS requests.
```
Comment (clue) [inline README.md:320]:
```suggestion
This library also allows you to send HTTP requests through an SSH proxy server.

In order to send HTTP requests, you first have to add a dependency for [ReactPHP's async HTTP client](https://github.com/reactphp/http#client-usage). You can then use this HTTP client in combination with the [`SshSocksConnector`](#sshsocksconnector) provided by this package. This allows you to send both plain HTTP and TLS-encrypted HTTPS requests like this:
```

File: README.md [hunk 2]
```diff
@@ -315,14 +315,20 @@ $connector->connect('tls://smtp.googlemail.com:465')->then(function (ConnectionI
...
+
+```php
+$proxy = new Clue\React\SshProxy\SshSocksConnector('me@localhost:22', $loop);
+
+$connector = new React\Socket\Connector($loop, array(
+    'tcp' => $proxy,
+    'dns' => false
+));
+
+$browser = new React\Http\Browser($loop, $connector);
```
Comment (clue) [inline README.md:330]:
```suggestion
$browser = new React\Http\Browser($loop, $connector);
$browser->get('https://example.com/')->then(function (Psr\Http\Message\ResponseInterface $response) {
    var_dump($response->getHeaders(), (string) $response->getBody());
});
```

File: README.md (hunk 1 above)
Comment (SimonFrings) [reply README.md]: Your version sounds better, thanks for the advice :+1:

**Comment by SimonFrings** [comment]
This pull request is outdated.
For continuation look into #21.

**Pull request closed without merging**